samples/hello_triangle_data/tri_fs.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+static const uint8_t s_tri_fs_bytecode_content[448] = {
```

Comment:
We want, in a comment, the string representation of the shader just above the bytecode here. Perhaps the compiler should output this directly into the header. sokol-shdc does this and it's greatly appreciated

Reply:
Sure. Preprocessed shader or original code?

Reply:
Probably preprocessed